handbook/customers.md

```diff
@@ -1,63 +1,63 @@
-# Customer engineering
+# Customers
 
 ## Customer success
 
 ### Scheduling meetings with customers
-To schedule an ad hoc meeting with a Fleet customer, use the ["Customer meeting" Calendly link](https://docs.google.com/document/d/1tE-NpNfw1icmU2MjYuBRib0VWBPVAdmq4NiCrpuI0F0/edit#heading=h.v47bs6uo0jpk).
+To schedule an ad hoc meeting (a video chat with multiple people or a non-scheduled meeting) with a Fleet customer, use the ["Customer meeting" Calendly link](https://docs.google.com/document/d/1tE-NpNfw1icmU2MjYuBRib0VWBPVAdmq4NiCrpuI0F0/edit#heading=h.v47bs6uo0jpk).
 
 ### Missed Zoom Meeting | Etiquette with Customers
 This is a tutorial on how to respond when someone from outside the company misses a call.
 
-#### 48 hours before a meeting
+#### 48 hours before a meeting 
```

Comment:
@mike-j-thomas, should "48-hours" be "forty-eight hours" or should we not write out numbers with hyphens?

Reply:
No change is needed. 48 hours is good 👍🏻